pyClient/testERCTokenMixing.py

```diff
@@ -132,12 +135,16 @@ def mint_token(token_instance, spender_address, deployer_address, token_amount):
         print("The allowance for Mixer from Bob is: ", allowance_mixer)
 
     # Bob approves the transfer
-    print("- Bob approves the transfer of 4 ETHToken to the Mixer")
-    tx_hash = approve(token_instance, bob_eth_address, mixer_instance.address, 4)
+    print("- Bob approves the transfer of ETHToken to the Mixer")
+    tx_hash = approve(
+        token_instance,
+        bob_eth_address,
+        mixer_instance.address,
+        zethTest.BOB_DEPOSIT_ETH)
     w3.eth.waitForTransactionReceipt(tx_hash)
     allowance_mixer = allowance(token_instance, bob_eth_address, mixer_instance.address)
     print("- The allowance for the Mixer from Bob is:", allowance_mixer)
-    # Bob deposits 4ETH split in 2 notes of denominations of 2ETh and 2ETH on the mixer
+    # Bob deposits ETHToken, split in 2 notes of denominations on the mixer
```

Comment:
`# Bob deposits ETHToken, split in 2 notes of denominations on the mixer` -> `# Bob deposits BOB_DEPOSIT_ETH, split in 2 notes of denominations BOB_SPLIT_1_ETH and BOB_SPLIT_2_ETH resp. on the mixer` ?